WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -543,8 +543,9 @@
         competition.update_attributes(start_date: "2011-12-04", end_date: "2011-12-05")
         get :post_announcement, params: { id: competition }
         post = assigns(:post)
-        expect(post.title).to eq "#{competition.name} on December 4 - 5, 2011 in #{competition.cityName}, #{competition.countryId}"
-        expect(post.body).to match(/in #{competition.cityName}, #{competition.countryId}\./)
+        country_name = Country.find(competition.countryId).name
```

Comment:
Can't you do `competition.country.name` here? Seehttps://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/models/competition.rb#L20.

Reply:
Ah, that seems easier indeed.
Maybe it's slower because you have to look for the country every time? Not that it would be an issue, I think.